CHANGELOG.md

```diff
@@ -33,6 +34,7 @@ maintainability of worker queries, and improve DB performance. ([PR](https://git
   and reliability at large scale. Workers older than v0.19.0 will remain supported
   until the release of v0.20.0, in accordance with
   [our worker/controller compatiblity policy](https://developer.hashicorp.com/boundary/docs/enterprise/supported-versions#control-plane-and-worker-compatibility).
+* ui: Update name radio option label & set toggle default to on. ([PR](https://github.com/hashicorp/boundary-ui/pull/2652)).
```

Comment:
That PR is associated with all the worker filter improvements, instead of referencing that one perhaps we should reference [this PR instead](https://github.com/hashicorp/boundary-ui/pull/2614). And change the description to something like `ui: Improve worker filter workflow for targets, vault credential-stores, and storage-buckets`

Reply:
Great idea 👍